diagnostic.md

```diff
@@ -51,19 +51,20 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
+ It is an absolute path because it starts at Users, which is the root directory. This is the full location of where Desktop is found.
```

Comment:
It is an absolute path, but the root directory is actually /.  /Users refers to the Users directory, which is contained in the root directory (/).